### ISSUE TITLE
Review

### ISSUE THREAD

**Issue by vr8hub** [brendanny/julia-peterkin_scarlet-sister-mary#1]
- [x] There is no semanticate commit. It would appear it is contained with the "Manual add additional semantics commit." As the Step by Step indicates (and as you did with typogrify), please keep the semanticate separate from any manual semantic changes done after it. Reviewers need to see them separate; we look closer at the manual one, and when the automated one is mixed in, it makes it hard to distinguish between the two and see what really needs to be looked at.
- [x] That same commit (5107165) has an editorial change in it (every one to everyone in ch. 7). As also covered in the Step by Step, we can't have editorial and non-editorial changes in the same commit. That change will have to be rebased out of that commit.
- [x] There were several instances of hear-say that were changed to hearsay by modernize spelling, but they're verbs (I hear say you did blah) in the text, not nouns. This is an instance of modernize spelling making a change that shouldn't be made; it doesn't happen very often, but it's why the Step by Step says that the changes it makes should be reviewed before being committed.
- [x] Broom sedge is what is in M-W, so we should standardize to that. Same for crab apple. Similarly, church-members is not in M-W, so we would standardize to church members. When standardizing things found by find-mismatched-dashes, we prefer to standardize to what is correct, even if it means changing both sets of things in the text. IOW, if we're going to change _something_, we might as well change _both_ things to what is correct.
- [x] It's impossible to say for sure, but from the looks of it, I'd say the entire dedication is small caps, so I don't think you need the spans. Also, we want to keep custom CSS separate from standard SE CSS, so the CSS to small cap the dedication should not be within the "centered dedication" standard CSS. (We generally put custom CSS at the bottom of local.css.)
- [x] The hymn in ch. 7 is indented in the second line in the scans, by more than just i1.
- [x] Typogrify adds a no-break space after "[Dd]e" because of it's use in, e.g., French names, to keep the rest of the name together with the "de." However, here de is used as vernacular, and so should not have the no-break space. They should be replaced with spaces.
- [x] "Good-standing" in the short description should be "good standing." (The former is again not in M-W.)
- [x] Should the long description begin "Set in _the_ post-Civil War…"? It sounds a bit odd to me without the "the."
- [x] Also, "was distinctive for her time in focusing on African-American experiences" is a bit awkward/ambiguous. It's hard to tell whether it's "was distinctive" "for her time in focusing …" or "was distinctive for her time" "in focusing …". IOW, was she distinctive because of the time she spent focusing, or was she distinctive for her time (era) because she was focusing? Maybe reword it to make it clear which of the two is your intent?
- [x] I'm not sure the description of the Pulitzer issue is correct. Both the Wiki article and [this](https://www.pulitzer.org/article/two-unlikely-heroines-modern-fiction) one on the Pulitzer site mention the jury chairman, but not the jury as a whole. The pulitzer.org article indicates that he spoke out of turn. Both articles say he resigned, but neither indicates a motive; neither does Peterson's Wiki article. Thus, "As a result…" is too strong a statement; something like "Subsequently…" or some such would be more accurate. One happened after the other, but we have no proof one caused the other.
- [x] As a whole we're not great about consistency on this, but _I_ try to be consistent when I use them or review books using them. :) It's The Online Distributed Proofreaders Canada Team. (This matches what is on Faded Page, and it matches, with the addition of "Canada," what we use for the PGDP team here, but Canada uses Proofreaders rather than Proofreading.)

**Comment by brendanny** [comment]
> There is no semanticate commit.

There were no changes made by `se semanticate` to commit. All semantics were manual.

> same commit ([5107165](https://github.com/brendanny/julia-peterkin_scarlet-sister-mary/commit/5107165356d5050fcb0b02d619096d322513b41a)) has an editorial change in it

Ah, yep. missed that in my rebase :(

> the entire dedication is small caps

At a glance, "To" looked like it wasn't but now that I'm looking again, the "o" does look small-caps. I'll get rid of the span.

> * Broom sedge is what is in M-W, so we should standardize to that. Same for crab apple.

I ultimately decided on usage on wikipedia and google's n-gram viewer: [broomsedge](https://books.google.com/ngrams/graph?content=broomsedge%2Cbroom+sedge%2C&year_start=1800&year_end=2022&corpus=en-US&smoothing=3) and [crabapple](https://books.google.com/ngrams/graph?content=crabapple%2Ccrab+apple&year_start=1800&year_end=2022&corpus=en-US&smoothing=3). But standardizing on M-W makes sense.

> * "Good-standing" in the short description

I hyphenated because "good standing" is acting as a compound modifier. I'll reword it 

> * It's The Online Distributed Proofreaders Canada Team

I actually looked around quite a bit for this. Examples in the SE corpus were pretty mixed and as the transcription does not capitalize "the online" nor "team", I defaulted to how they refer to themselves on their [website](https://www.pgdpcanada.net/c/default.php): "Distributed Proofreaders Canada".

**Comment by brendanny** [comment]
> several instances of hear-say that were changed to hearsay

should I choose "hear-say" or "hear say"

**Comment by brendanny** [comment]
similarly to good-standing, crab apple is used as a compound modifier in both places
> crab[-]apple thickets
> crab[-]apple blossoms

**Comment by vr8hub** [comment]
> There were no changes made by se semanticate to commit. All semantics were manual.

There would have been if you had run semanticate before the manual changes.

> Examples in the SE corpus were pretty mixed

As I said, we haven't been good about that. :) But I try to standardize everything I touch, and, again, it is consistent both with what's on their website (capitalized or not) and what we use for PGDP US.

> should I choose "hear-say" or "hear say"

I would probably choose the latter, but I don't think it matters. If modernize spelling hadn't changed it, I wouldn't have, either.

**Comment by brendanny** [comment]
Okay, I think these should all be taken care of now. Thanks for the thorough review Vince!

Let me know if I need to work on the description more
